terraform/functions.tf

```diff
@@ -357,23 +357,16 @@ module "lambda_function-processValidationJson" {
   policy_jsons                      = local.lambda_default_execution_policies
   attach_policy_statements          = true
   policy_statements = {
-    AllowDownloadExcelObjects = {
+    AllowDownloadAndDeleteJsonObjects = {
       effect = "Allow"
       actions = [
         "s3:GetObject",
         "s3:HeadObject",
+        "s3:DeleteObject",
```

Comment:
Was this intended to be `s3:PutObject`?

Reply:
no, as mentioned in the latest comment. This function needs to only be able to `get` and `delete` the `.json` files